### PR TITLE
New version

### PR DESCRIPTION
**Pull request by strizr**
### Motivation

Updated styles based on new Kontent app design + new version of Bynder's Compact view added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ You will need to add the custom element to a content type filling in the hosted
 
 ```json
 {
-  "bynderUrl": "<YOUR BYNDER URL>",
-  "previewDerivative": "webimage",
-  "webDerivative": "webimage"
+  "bynderUrl": "https://<YOUR BYNDER URL>",
+  "previewDerivative": "thumbnail",
```
Comment (Simply007) [inline README.md:63]: Why previewDerivative changed?

File: README.md [hunk 2]
```diff
@@ -78,23 +78,17 @@ Example output:
...
+    "bynderUrl":"https://test.getbynder.com/media/?mediaId=36AAB6D3-7DFE-41AB-A0B95A826D4Ctest",
+    "updatedAt":"2021-01-20T17:40:14Z",
+    "description" : "test image"
   }
 ]
 ```
 
-## Live site implementation sample
-
-If you want to see live site example of Bynder images displayed on the live site, browse to a [deployed sample site](https://kentico-cloud-sample-app-react-bynder.surge.sh/en-us/articles/cf106f4e-30a4-42ef-b313-b8ea3fd3e5c5).
```
Comment (Simply007) [inline README.md:88]: If we don't link Bynder anywhere, we can remove the branch from the repository.

File: README.md [hunk 3]
```diff
@@ -78,23 +78,17 @@ Example output:
...
-
-See source code of the sample site implementation [here](https://github.com/Kentico/kontent-sample-app-react/commit/b4ae153a1acde2feb10b865dbf338eaca84f3c35)
-
-
 ## Contributors
 We have collected notes on how to contribute to this project in [CONTRIBUTING.md](CONTRIBUTING.md).
 
-Originally created by [@kenticomartinh](https://github.com/kenticomartinh)
```
Comment (Simply007) [inline README.md:96]: :( - are we going to add a contributors widget?

File: index.html [hunk 4]
```diff
@@ -6,167 +6,25 @@
   <title>Bynder media selector</title>
   <!-- Kontent Styling -->
   <link rel="stylesheet" type="text/css"
-    href="https://kentico.github.io/kontent-custom-element-samples/shared/custom-element.css">
+    href="https://kentico.github.io/kontent-custom-element-samples/shared/custom-element-v2.css">
```
Comment (Simply007) [inline index.html:9]: do we want to rely on the deployed version?

**Review by Simply007 — COMMENTED** [review]
Just walked through the code changes (haven't tested the actual functionality).

File: README.md (hunk 1 above)
Comment (strizr) [reply README.md:63]: the default value is thumbnail

File: README.md (hunk 2 above)
Comment (strizr) [reply README.md:88]: yes, we can!

File: index.html (hunk 4 above)
Comment (strizr) [reply index.html:9]: yes, we want!

**Pull request merged**